youtube_transcript_api/_api.py

```diff
@@ -1,11 +1,15 @@
 import requests
+try:
+    import http.cookiejar as cookiejar
+except ImportError:
+    import cookielib as cookiejar
 
 from ._transcripts import TranscriptListFetcher
 
 
 class YouTubeTranscriptApi():
     @classmethod
-    def list_transcripts(cls, video_id, proxies=None):
+    def list_transcripts(cls, video_id, proxies=None, cookies=None):
```

Comment:
Please update the docstrings of `list_transcript`, `get_transcript` and `get_transcripts` accordingly.